format/spec.md

```diff
@@ -1754,6 +1754,10 @@ Snapshot summary can include metrics fields to track numeric stats of the snapsh
 | **`engine-name`**        | "spark"    | Name of the engine that created the snapshot                    |
 | **`engine-version`**     | "3.5.4"    | Version of the engine that created the snapshot                 |
 
+### Encoding of `current-snapshot-id`
+
+The Java implementation writes `-1` for  "no current snapshot" with V1 and V2 tables and considers this equivalent to missing/null. This has never been formalized in the spec but for compatibility other implementations can accept `-1` as `null`. The Java implementation will no longer write `-1` and will use null for "no current snapshot for all tables with a version greater than or equal to V3
```

Comment:
Puncutation nits: 

- extra space between `for "no current snapshot"`
- comma before 'but' (second is complete sentence)
- period at end
- should we be consistent whether `null` is code-denoted or not

Reply:
```suggestion
The Java implementation writes `-1` for "no current snapshot" with V1 and V2 tables and considers this equivalent to omitted or `null`. This has never been formalized in the spec but for compatibility, other implementations can accept `-1` as `null`. The Java implementation will no longer write `-1` and will use `null` for "no current snapshot" for all tables with a version greater than or equal to V3.
```

Reply:
Looks great, thanks !